### PR TITLE
Update with `graalvmCompatible`  property

### PR DESCRIPTION
**Pull request by AzeemMuzammil**
# Description

$title.

Related Pull Requests (remove if not relevant)
- Pull request 1
- Pull request 2

One line release note: 
- One line describing the feature/improvement/fix made by this PR 

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:
* Ballerina Version:
* Operating System:
* Java SDK: 

# Checklist:

### Security checks
 - [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? 
 - [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -1,4 +1,4 @@
 org.gradle.caching=true
 group=io.ballerinax
 version=7.2.0
-ballerinaLangVersion=2201.4.0-20230124-005500-f19b94e2
+ballerinaLangVersion=2201.7.0-20230619-175900-bb4e4544
```
Comment (TharmiganK) [inline gradle.properties:4]: Ballerina Lang has released the stable version - `2201.7.0`. We can use that here.
```suggestion
ballerinaLangVersion=2201.7.0
```

File: salesforce/Ballerina.toml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 [package]
-distribution = "2201.3.0"
+distribution = "2201.7.0"
```
Comment (TharmiganK) [inline salesforce/Ballerina.toml:2]: I think we need to update the [`Package.md`](https://github.com/ballerina-platform/module-ballerinax-sfdc/blob/master/salesforce/Package.md) as well

File: salesforce/Ballerina.toml [hunk 3]
```diff
@@ -1,5 +1,5 @@
...
 org = "ballerinax"
 name = "salesforce"
 version = "7.2.0"
```
Comment (TharmiganK) [inline salesforce/Ballerina.toml:5]: We need to bump to the next minor version with the distribution version update. But this version is still outdated, I think the latest released version is `7.4.0`. But why it is not updated here?

Comment (AzeemMuzammil) [reply salesforce/Ballerina.toml:5]: Lemme update the version,

File: gradle.properties [hunk 4]
```diff
@@ -1,4 +1,4 @@
 org.gradle.caching=true
 group=io.ballerinax
-version=7.2.0
-ballerinaLangVersion=2201.4.0-20230124-005500-f19b94e2
+version=7.4.1
```
Comment (NipunaRanasinghe) [inline gradle.properties:3]: Any specific reason to go for `7.4.1` instead of `7.2.1`?

Comment (AzeemMuzammil) [reply gradle.properties:3]: already 7.4.0 is released and it's not updated in the master branch

**Review by NipunaRanasinghe — APPROVED** [review]

**Review by TharmiganK — APPROVED** [review]

**Review by NipunaRanasinghe — APPROVED** [review]

**Pull request merged**